### ISSUE TITLE
adaptive loss weight

### ISSUE THREAD

**Issue by fereenwong** [CPFLAME/centerX#3]
Hi, I had a similar training problem in another task. I want to try the adaptive loss weight strategy. I am wondering how to select COMMUNISM.CLS_LOSS, COMMUNISM.WH_LOSS and COMMUNISM.OFF_LOSS? Can you share the experience with me? Anyway, thanks a lot.

**Comment by CPFLAME** [comment]
I just follow the normal training loss at suitable learning rate:
for coco(80 class):
COMMUNISM.CLS_LOSS  = 1.5
COMMUNISM.WH_LOSS = 0.3
COMMUNISM.OFF_LOSS = 0.1
for crowd human(1 class):
COMMUNISM.CLS_LOSS  = 1.2
COMMUNISM.WH_LOSS = 0.4
COMMUNISM.OFF_LOSS = 0.1

I think the more training class, the larger CLS_LOSS/WH_LOSS.
You can modify it in your custom dataset
